connectors/sources/onedrive.py

```diff
@@ -305,12 +331,30 @@ async def paginated_api_call(self, url, params=None, fetch_size=FETCH_SIZE):
                 break
 
     async def list_users(self):
```

Comment:
Consider make it optional to include groups
```suggestion
    async def list_users(self, include_groups=False):
```